test/e2e/devconsole.operatorsource.cr.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: devconsole-operator
+  namespace: openshift-operators
+spec:
+  type: appregistry
+  endpoint: https://quay.io/repository/redhat-developer/devconsole-operator
```

Comment:
Does this actually work like an app registry?

Reply:
It allowed the subscription to be created - but you are correct - it's not an app registry. Will retry after having created an app registry in Quay.